### PR TITLE
add gce-staging-2

### PR DESCRIPTION
**Pull request by igorwwwwwwwwwwwwwwwwwwww**
to be used for nat load testing and other things that we do not want to impact staging-1

### CODE REVIEW

File: gce-staging-2/main.tf [hunk 1]
```diff
@@ -0,0 +1,51 @@
+variable "env" { default = "staging-2" }
```
Comment (meatballhat) [inline gce-staging-2/main.tf]: This should be `default = "staging"`

File: gce-staging-2/main.tf [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+variable "gce_bastion_image" { default = "eco-emissary-99515/bastion-1478778272" }
+variable "gce_gcloud_zone" {}
+variable "gce_heroku_org" {}
+variable "gce_nat_image" { default = "eco-emissary-99515/nat-1478778271" }
+variable "gce_hashistack_server_image" { default = "eco-emissary-99515/hashistack-server-1480351044" }
+variable "gce_worker_image" { default = "eco-emissary-99515/travis-worker-1480649763" }
+variable "github_users" {}
+variable "index" { default = 1 }
```
Comment (meatballhat) [inline gce-staging-2/main.tf]: This should be `default = 2`

Comment (igorwwwwwwwwwwwwwwwwwwww) [inline gce-staging-2/main.tf]: ah yes, that is a nicer way of doing it. thanks! :)

**Comment by igorwwwwwwwwwwwwwwwwwwww** [comment]
@meatballhat applied suggested change!

**Pull request merged**